extensions/tokengate-src/package.json

```diff
@@ -9,6 +9,9 @@
     "build": "vite build",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "engines": {
+    "node": ">=14.13.1 <17.0.0"
```

Comment:
#6 removes these engine specifications so we may want to either remove them totally or move this to the root package